.circleci/scripts/trigger-beta-build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+# => Version-v10.24.1
+version="${CIRCLE_BRANCH/Version-v/}"
+current_commit_msg=$(git show -s --format='%s' HEAD)
+printf '%s\n' "Creating the prod beta build for $version under $current_commit_msg"
+
+if [[ $current_commit_msg =~ Version[-[:space:]](v[[:digit:]]+.[[:digit:]]+.[[:digit:]]+[-]beta.[[:digit:]]) ]]
```

Comment:
In the regexes, there is a dash between the version and the v, and here too. However, in other bash if statements, you don't have the dash. Are these all checking for the same stuff, or are there multiple patterns?

Reply:
good catch. so all the messages are consistently as `Version v10.2.1-beta` and will be auto generated.